### DISCUSSION TITLE
What is the best way to use a custom, fine-tuned Whisper model with this library?

### DISCUSSION THREAD

**Discussion by tgvarner** [openai/whisper#2242] [Q&A]
I want to use a custom, fine-tuned model with this library. What is the cleanest solution to do so, such that I can call :

```whisper.load_model("/path/to/custom/model")```

How should I go about storing and converting the files (https://huggingface.co/openai/whisper-large-v3/tree/main) stored in the HG repository to load the model via the above function?

I found this thread: https://github.com/openai/whisper/discussions/1170, but I didn't find it helpful. Has anyone figured out a clean and way to do this?

Thanks!

**Comment by phineas-pta** [comment]
either see #830

or use `transformers` package from `huggingface`
